src/igvfd/schemas/sequence_file.json

```diff
@@ -354,8 +354,19 @@
                 "enum": [
                     "Read 1",
                     "Read 2",
-                    "Barcode index"
-                ]
+                    "Barcode index",
+                    "Barcode forward",
+                    "UMI",
+                    "Barcode reverse"
+                ],
+                "enum_descriptions": {
+                    "Read 1": "Read name associated with the single-cell uniform processing pipeline.",
+                    "Read 2": "Read name associated with the single-cell uniform processing pipeline.",
+                    "Barcode index": "Read name associated with the single-cell uniform processing pipeline.",
+                    "Barcode forward": "Read name associated with the MPRA uniform processing pipeline.",
+                    "UMI": "Read name associated with the MPRA uniform processing pipeline.",
+                    "Barcode reverse": "Read name associated with the MPRA uniform processing pipeline."
```

Comment:
Would it be possible to add something to the definitions for forward and reverse just so they are distinct? (even though it's pretty obvious, but I think it's better to include it to be thorough.)